### PR TITLE
Remove Clique

### LINKED ISSUE #1034: Remove clique from docs
* remove https://besu.hyperledger.org/en/stable/HowTo/Configure/Consensus-Protocols/Clique/
* remove tutorials https://besu.hyperledger.org/en/stable/Tutorials/Private-Network/Create-Private-Clique-Network/
* remove https://besu.hyperledger.org/en/stable/HowTo/Configure/Consensus-Protocols/Clique/#migrate-from-clique-to-another-consensus-protocol 
* change permissioning tutorial from Clique to IBFT2 https://besu.hyperledger.org/en/stable/Tutorials/Permissioning/Create-Permissioned-Network/
* Also mentioned here https://besu.hyperledger.org/en/stable/Concepts/Consensus-Protocols/Comparing-PoA/

- [ ] Missing content
- [x] Outdated content
- [ ] Wrong content
- [ ] Confusing or misleading content
- [ ] Other

## Change suggestion

Change any tutorials to use QBFT/IBFT2

### PR DESCRIPTION
**Pull request by rolandtyler**
Signed-off-by: Roland Tyler <roland.tyler@consensys.net>

## Pull request checklist

Use the following list to make sure your PR fits the Besu documentation quality standard.

### Before creating the pull request

Make sure that:

- [x] [All commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [x] You've read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Documentation).
- [x] You've [previewed your changes locally](https://wiki.hyperledger.org/display/BESU/Preview+the+documentation).

## Describe the change
Remove Clique tutorials and references

## Issue fixed
Fixes #1034

## Impacted parts

For content changes:

- [x] Documentation content
- [x] Documentation page organization

For tool changes:

- [ ] CircleCI workflow
- [ ] Build and QA tools (for example, lint or vale)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] Read the Docs configuration
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [x] You've fixed any issues raised by the tests.
- [x] You've [previewed your changes on Read the Docs](https://wiki.hyperledger.org/display/BESU/Preview+the+documentation)
  and added a [preview link](#preview).

### CODE REVIEW

File: docs/Tutorials/Permissioning/Getting-Started-Onchain-Permissioning.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ description: Setting up and using Hyperledger Besu onchain permissioning
 The following steps describe bootstrapping a permissioned network using a Hyperledger Besu
 node and a development server to run the permissioning management dapp.
 
-This tutorial configures permissioning on a [Clique proof of authority (PoA)] network.
+This tutorial configures permissioning on a [IBFT 2.0 proof of authority (PoA)] network.
```
Comment (macfarla) [inline docs/Tutorials/Permissioning/Getting-Started-Onchain-Permissioning.md:10]: The changes to this tutorial to change from Clique to IBFT2 - could you make that a separate PR?

File: docs/Tutorials/Permissioning/Upgrade-Permissioning-Contract.md [hunk 2]
```diff
@@ -162,7 +162,7 @@ version 2.
 
 !!! example
     ```cmd
-        besu --data-path=data --genesis-file=../cliqueGenesis.json --permissions-accounts-contract-enabled --permissions-accounts-contract-address "0x0000000000000000000000000000000000008888" --permissions-nodes-contract-enabled  --permissions-nodes-contract-address "0x4E72770760c011647D4873f60A3CF6cDeA896CD8" --permissions-nodes-contract-version=2 --rpc-http-enabled --rpc-http-cors-origins="*" --rpc-http-api=ADMIN,ETH,NET,PERM,CLIQUE --host-allowlist="*"
```
Comment (macfarla) [inline docs/Tutorials/Permissioning/Upgrade-Permissioning-Contract.md:165]: ditto - include this in the permissioning tutorial change PR

File: docs/Tutorials/Permissioning/Create-Permissioned-Network.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ description: Hyperledger Besu create a permissioned network
 # Create a permissioned network
 
 The following steps set up a permissioned network with local node and account permissions. The network
-uses the [Clique proof of authority consensus protocol].
+uses the [IBFT 2.0 proof of authority consensus protocol].
```
Comment (macfarla) [inline docs/Tutorials/Permissioning/Create-Permissioned-Network.md:8]: ditto here

**Review by macfarla — COMMENTED** [review]
Can you make a separate PR for changing the tutorials over to IBFT2? Still not sure if we can totally remove clique but we can remove it from tutorials.

**Review by shemnon — CHANGES_REQUESTED** [review]
Changing tutorials over to QBFT or IBFT2 is fine.

However Clique is still a mandated protocol in the current as well as the pending draft of the EEA Client Sepcification (https://entethalliance.github.io/client-spec/spec.html#sec-consensus-sublayer).  Since Besu was founded on EEA compatibility we have a requirement that clique must be implemented.  Scrubbing documentation runs counter to that obligation.

When The EEA spec removes clique as a MUST then we can change code and scrub documentation, until then we should continue to document it.

**Comment by rolandtyler** [comment]
Closing per comment on EEA requirement

**Pull request closed without merging**